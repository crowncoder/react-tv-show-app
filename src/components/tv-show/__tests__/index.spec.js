import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import EnzymeAdapter from 'enzyme-adapter-react-16'
import TvShowInfo from '../';
import ProfileInfo from '../profile-info';
import EpisodeList from '../episode-list';


Enzyme.configure({ adapter: new EnzymeAdapter() });

describe('<TvShowInfo /> unit test', () => {
    const getWrapper = () => shallow(
            <TvShowInfo />
    );

    it('should render correctly', () => {
        const wrapper = getWrapper();
        expect(wrapper.find(ProfileInfo)).toHaveLength(1);
        expect(wrapper.find(EpisodeList)).toHaveLength(1);
        expect(wrapper).toMatchSnapshot();
    });
});