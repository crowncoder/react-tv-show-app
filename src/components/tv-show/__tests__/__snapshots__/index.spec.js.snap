// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TvShowInfo /> unit test should render correctly 1`] = `
<Fragment>
  <Memo() />
  <Memo() />
</Fragment>
`;
